bootstrap4/.js/src/main/scala/io/udash/bootstrap/utils/BootstrapStyles.scala

```diff
@@ -672,3 +597,94 @@ object BootstrapStyles {
+  object ResponsiveBreakpoint extends AbstractValueEnumCompanion[ResponsiveBreakpoint] {
+    final val All: Value = new ResponsiveBreakpoint("")
+    final val Small: Value = new ResponsiveBreakpoint("-sm")
+    final val Medium: Value = new ResponsiveBreakpoint("-md")
+    final val Large: Value = new ResponsiveBreakpoint("-lg")
+    final val ExtraLarge: Value = new ResponsiveBreakpoint("-xl")
+    final val Print: Value = new ResponsiveBreakpoint("-print")
+  }
+
+  final class Side(val classMarker: String, val longClassMarker: String)(implicit enumCtx: EnumCtx) extends AbstractValueEnum
+  object Side extends AbstractValueEnumCompanion[Side] {
+    final val All: Value = new Side("", "")
+    final val Top: Value = new Side("t", "-top")
+    final val Bottom: Value = new Side("b", "-bottom")
+    final val Left: Value = new Side("l", "-left")
+    final val Right: Value = new Side("r", "-right")
+    /**
+      * This value is not always supported by Bootstrap.
+      * It works with spacing, but you cannot use it with borders methods. <br/>
+      * More: <a href="https://getbootstrap.com/docs/4.1/utilities/spacing/">Spacing Docs</a> and
+      * <a href="https://getbootstrap.com/docs/4.1/utilities/borders/">Borders Docs</a>
+      */
+    final val X: Value = new Side("x", "")
+    /**
+      * This value is not always supported by Bootstrap.
+      * It works with spacing, but you cannot use it with borders methods. <br/>
+      * More: <a href="https://getbootstrap.com/docs/4.1/utilities/spacing/">Spacing Docs</a> and
+      * <a href="https://getbootstrap.com/docs/4.1/utilities/borders/">Borders Docs</a>
+      */
+    final val Y: Value = new Side("y", "")
+  }
+
+  final class Align(val classMarker: String)(implicit enumCtx: EnumCtx) extends AbstractValueEnum
+  object Align extends AbstractValueEnumCompanion[Align] {
+    final val Left: Value = new Align("-left")
+    final val Right: Value = new Align("-right")
+    final val Center: Value = new Align("-center")
+  }
+
+  final class Color(val classMarker: String)(implicit enumCtx: EnumCtx) extends AbstractValueEnum
+  object Color extends AbstractValueEnumCompanion[Color] {
+    final val Primary: Value = new Color("-primary")
+    final val Secondary: Value = new Color("-secondary")
+    final val Success: Value = new Color("-success")
+    final val Danger: Value = new Color("-danger")
+    final val Warning: Value = new Color("-warning")
+    final val Info: Value = new Color("-info")
+    final val Light: Value = new Color("-light")
+    final val Dark: Value = new Color("-dark")
+    final val White: Value = new Color("-white")
+    /** This value is only supported in buttons. */
+    final val Link: Value = new Color("-link")
+  }
+
+  final class Size(val classMarker: String)(implicit enumCtx: EnumCtx) extends AbstractValueEnum
+  object Size extends AbstractValueEnumCompanion[Size] {
```

Comment:
`xl` missing
Modal accepts xl size: [Modal](https://getbootstrap.com/docs/4.3/components/modal/#optional-sizes)

Reply:
It doesn't on Boostrap 4.1, migration will be another thing.